tests/test_csc.py

```diff
@@ -145,12 +144,13 @@ async def test_lifecycle_behavior(self):
             )
             self.assertEqual(state.summaryState, salobj.State.STANDBY)
 
-    async def test_bad_simulation_mode(self):
-        """Test to ensure bad simulation modes raise
-        """
+    async def xtest_bad_simulation_mode(self):
```

Comment:
Please restore this test.

Also: it should not need to be asynchronous (though it's fine with me if you prefer to leave the "async" in).

Reply:
Done.

Reply:
Done.